Makes the generated documentation reproducible (no time stamps).
Should be offered to upstream.

--- wireshark-3.6.1/cmake/modules/FindAsciidoctor.cmake
+++ wireshark-3.6.1/cmake/modules/FindAsciidoctor.cmake
@@ -42,6 +42,7 @@
         # --failure-level=WARN
         # --trace
         --quiet
+        --attribute reproducible
         --attribute build_dir=${CMAKE_CURRENT_BINARY_DIR}
         --require ${CMAKE_SOURCE_DIR}/docbook/asciidoctor-macros/ws_utils.rb
         --require ${CMAKE_SOURCE_DIR}/docbook/asciidoctor-macros/commaize-block.rb
